pkg/machine/shim/host.go

```diff
@@ -543,6 +569,17 @@ func Start(mc *vmconfigs.MachineConfig, mp vmconfigs.VMProvider, dirs *machineDe
 		}
 	}
 
+	if mp.VMType() == machineDefine.WSLVirt && mc.Ansible != nil {
+		user := "user"
+		if mc.HostUser.Rootful {
+			user = "root"
+		}
+
```

Comment:
I'm realizing now this is a poor assumption. I should keep track of the user the playbook is originally created under. The rootfulness may change.